package.json

```diff
@@ -1,85 +1,34 @@
 {
-  "name": "@jupiterone/integration-sdk",
-  "version": "3.0.3",
-  "description": "The SDK for developing JupiterOne integrations",
-  "main": "src/framework/index.js",
-  "types": "src/framework/index.d.ts",
-  "repository": "git@github.com:JupiterOne/integration-sdk.git",
-  "author": "JupiterOne <dev@jupiterone.io>",
-  "license": "MPL-2.0",
-  "engines": {
-    "node": "10.x || 12.x || 14.x"
-  },
-  "publishConfig": {
-    "access": "public"
-  },
-  "bin": {
-    "j1-integration": "./bin/j1-integration"
-  },
+  "name": "root",
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "lint": "eslint . --cache --fix --ext .ts,.tsx",
-    "format": "prettier --write '**/*.{ts,js,json,css,md,yml}'",
-    "type-check": "tsc --noEmit",
+    "lint": "eslint --ext .ts,.tsx,.js,.jsx .",
+    "type-check": "tsc -b packages/*",
```

Comment:
Experimenting with tsc's new composite builds. This setup is different from the typical `--noEmit` flow we usually do because code actually gets transpiled. The main benefit of this new change is that typescript can store some metadata and do faster incremental builds.

Reply:
Ran into issues with this on windows... For now we'll have to specify each package. I'll work on a script to make this less painful as more packages get introduced.